docs/conf.py

```diff
@@ -91,7 +91,10 @@
 # Add and use Pylons theme
 html_theme = 'pylons'
 html_theme_path = pylons_sphinx_themes.get_html_themes_path()
-html_theme_options = dict(github_url='http://github.com/Pylons/waitress')
+html_theme_options = dict(
+    github_url='http://github.com/Pylons/waitress',
+    canonical_url='http://docs.pylonsproject.org/projects/waitress/en/latest/'
```

Comment:
Looking at the source for http://waitress.readthedocs.io/en/latest/, that link is already present.